pyscf/pbc/gto/cell.py

```diff
@@ -909,7 +922,7 @@ def make_kpts(cell, nks, wrap_around=WRAP_AROUND, with_gamma_point=WITH_GAMMA,
         kpts = libkpts.make_kpts(cell, kpts, space_group_symmetry, time_reversal_symmetry, symmorphic)
     return kpts
 
-def get_uniform_grids(cell, mesh=None, **kwargs):
+def get_uniform_grids(cell, mesh=None, wrap_around=True):
```

Comment:
I think setting `wrap_around` default as `True` may break other codes following the previous convention.

Reply:
When quantities and integrals are perfectly converged, there's no difference if the grids are inside the unit cell or wrapped around against the point (0,0,0) . If quantities are barely converged, it may have small difference to the other convention. This setting favors the systems in which atoms are placed near the edge of the cell, e.g. a 2D slab.